settings/test.py

```diff
@@ -22,4 +22,5 @@
     }
 }
 
+DEV = False
```

Comment:
We don't need a new variable `DEV`. 

`settings.DEBUG = True` means dev environment only.

Reply:
Please check the condition  https://github.com/Cloud-CV/EvalAI/pull/1792/files#diff-aa6d983093bc40e4ce5d8205f511db8dR78

Reply:
This variable will cause confusion for no reason. We already have the `TEST` variable which you can make use of in the code snippet.